quic/s2n-quic-events/src/parser.rs

```diff
@@ -282,6 +283,25 @@ impl Struct {
                         }
                     ));
 
+                    // Metrics is only connection-level events
+                    output.metrics_fields.extend(quote!(
+                        pub #counter: #counter_type,
```

Comment:
Let's remove `pub` - the application shouldn't need to access it, at least at this point
```suggestion
                       #counter: #counter_type,
```